corehq/util/couch.py

```diff
@@ -245,4 +245,10 @@ def _iter_update(doc_ids, try_num):
                "change or remove the '_id' field?".format(fn.__name__) +
                ", ".join(results.error_ids))
         raise IterUpdateError(results, msg)
+
+    if verbose:
+        print "couldn't find {} docs".format(len(results.not_found_ids))
+        print "ignored {} docs".format(len(results.ignored_ids))
+        print "deleted {} docs".format(len(results.deleted_ids))
+        print "updated {} docs".format(len(results.updated_ids))
```

Comment:
You should just be able to do this:
```
logger = logging.getLogger(__name__)
logger.setLevel(logging.DEBUG)
```
And as long as you have a handler that's level is also set to DEBUG then the output will get printed